_posts/2015-06-26-ux-part-2.md

```diff
@@ -0,0 +1,108 @@
+
+<img src="/images/workflow.png" alt="A diagram of the workflow of OME User Centred Design" style="width: 600px;"/>
+
+*Diagram of workflow of User Centred Design.*
+
+##Requirements Gathering
+
+Most of the UX team work starts with requirements in some form or another. This can range from a formal request through our forums or mailing lists to informal requests from users at meetings or even people sidling up to Jason in the pub and telling him they need OMERO to do the next big thing.
+
+The requirement can often be a single use-case from a bench scientist, ranging from a simple task through to a large, complex workflow. This can be entirely new or a request for enhancement (RFE) of an existing workflow or piece of functionality. Commonly, the requirement is specific to a relatively small number of users, and they are often very busy people and may be located in many different parts of the world.
+
+This means it can be a challenge spending enough time with a wide enough range of users to ensure the requirements are not too narrow and generalise to be applicable to the widest range of users and use cases as possible. However we generally find that once people are on board they are very generous with their time and expertise.
+
+##Client Group Meetings
+
+Our workflow revolves around our Client Group meetings where members of the team present requirements, talk through issues and present the results of any scoping for feedback. And this is where we usually relive the classic tussle between conceptual and functional designs.
+
+The traditional approach is that the conceptual design stage it’s all blue-sky, open-ended, ideal-world thinking. The idea is not to close off any avenues at an early stage and instead try and think about what the system should really do and how it should really work. All fluffy, cuddly and cute. 
+
+<img src="/images/conceptual-function-design.png" alt="Cartoon showing the ideal world separation of conceptual and functional design." style="width: 600px;"/>
+
+*The ideal of Conceptual to Functional design.*
+
+Then you traditionally progress to functional design which is the reality of how it will all be done – the nuts and bolts of it. Well that is the traditional view ...
+
+... the reality is usually more like this - but it shouldn’t stop you trying!
+
+<img src="/images/the-reality.png" alt="Cartoon showing the mash-up that usually results with conceptual and functional design." style="width: 500px;"/>
+
+*The reality of conceptual and functional design separation!*
+
+Despite this, keeping that open mind and idealism going for as long as possible in the process is very productive.
+
+We are often faced with the prospect of a fairly daunting amount of change to the system to address some of the more complex requirements requested. The UX team has found it works well to see if there is a way of chunking the proposed changes it into smaller steps. 
```

Comment:
"chunking the proposed changes it into smaller steps" doesn't read well to me, maybe "splitting the proposed changes into smaller steps" ?